test/api/assignments_test.rb

```diff
@@ -30,8 +30,9 @@ def test_api_accepts_submission_attempt
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
         solution = { "THE CODE" => "ruby/one/code.rb" }
-        post '/user/assignments',  { key: alice.key, code: 'THE CODE', path: 'ruby/one/code.rb', 
-          'language' => 'ruby','problem' => 'one', solution: solution }.to_json
+        post '/user/assignments',  { key: alice.key, code: "THE CODE", path: "ruby/one/code.rb",
```

Comment:
Line is too long. [96/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.